modules/json/json-parser.c

```diff
@@ -109,8 +109,8 @@ json_parser_process_single(struct json_object *jso,
       break;
     case json_type_int:
       parsed = TRUE;
-      g_string_printf(value, "%i",
-                      json_object_get_int(jso));
+      g_string_printf(value, "%ld",
```

Comment:
I think we should use `PRId64` or `G_GINT64_FORMAT`.

Reply:
`PRId64` it is.